.drone.yml

```diff
@@ -4656,6 +4656,55 @@ volumes:
 - name: dockersock
   temp: {}
 
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/misc.go:30
+################################################
+
+kind: pipeline
+type: kubernetes
+name: publish-apt-rfd0058-repos
+trigger:
+  event:
+    include:
+    - promote
+  target:
+    include:
+    - production
+  repo:
+    include:
+    - gravitational/*
+workspace:
+  path: /go
+clone:
+  disable: true
+steps:
+- name: Publish debs to APT repos
+  image: ubuntu:20.04
+  commands:
+  - cd /go/src/github.com/gravitational/teleport/build.assets/tooling
+  - export VERSION="v`cat /go/build/CURRENT_VERSION_TAG_GENERIC.txt`"
+  - export RELEASE_CHANNEL="stable"
+  - go run ./cmd/build-apt-repos -bucket "$APT_S3_BUCKET" -artifact-major-version
+    "$VERSION" -artifact-release-channel "$RELEASE_CHANNEL" -artifact-path "/go/artifacts/"
+    -log-level 4
+  environment:
+    APT_S3_BUCKET:
+      from_secret: APT_REPO_RFD0058_AWS_S3_BUCKET
```

Comment:
Nit: Do not include RFD number.

Reply:
Any suggestions on name here? This differentiates the variable(s) for RFD 0058 publishing pipelines from the variables for the current pipelines.

Reply:
What's the context of this choice? Will the current pipelines coexist with with the changes introduced here, or will they be out-and-out replaced? 

If there is some overlap, but the old pipelines will be replaced soon, can _their_ secrets be renamed `LEGACY_XXXX` to differentiate until the switchover?

Reply:
Updated to use "NEW" instead of RFD0058 everywhere. We chose not to rename the other secrets as "LEGACY" due to an increased risk of breaking things.